RELEASE_NOTES.md

```diff
@@ -8,16 +8,19 @@
 [//]: # ( # Deprecated)
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
-## Added
-- Dual-branded card flow: shoppers can now select their card's brand when two brands are detected.
-- Ability to store BCMC cards.
-- Support for new Activity Result API.
-- Ability to stop observing components.
-- 3DS2 SDK version in `CardComponent`'s output.
-- Read list of supported card networks for Google Pay from the payment methods API response.
-
 ## Fixed
-- Crash caused by having stored payment methods none of which is Ecommerce.
-- Google Pay Component will not include `TotalPrice` in its output, if `TotalPriceStatus` is set to `NOT_CURRENTLY_KNOWN`.
-- Issue in Drop-in when multiple payment methods have the same type.
-- Missing default BCMC configuration in Drop-in.
+- Make sure you explicitly set your LIVE environment on release builds when initializing Drop-in; `DropInConfiguration` is now initialized with `Environment.TEST` by default instead of `Environment.EUROPE`.
```

Comment:
I think this should be on it's own session. Like

## WARNING
The default environment for `DropInConfiguration` is now `Environment.TEST`. Make sure you explicitly set your LIVE environment on release builds when initializing